public/content/lessons/2019/windmills/index.mdx

```diff
@@ -1,9 +1,232 @@
 ---
 title: The unexpectedly hard windmill question (2011 IMO, Q2)
-description: Problem 2 from the 2011 IMO
+description: Problem 2 from the 2011 IMO, a seemingly easy question that turned out to be ridiculously difficult
 date: 2019-08-04
 video: M64HUIJFTZM
 source: _2019/windmill.py
 credits:
-- Lesson by Grant Sanderson
+    - Lesson by Grant Sanderson
+    - Text Adaptation by Vivek Verma
 ---
+
+## The International Math Olympiad (IMO)
+
+Every year, more than 100 countries send 6 of their brightest teenagers, or the occasional pre-pubescent prodigy, to represent them in the International Math Olympiad, commonly known as the IMO.
+
+<Figure
+    image="imo.png"
+/>
+
+Considering that each country has its own elaborate system of contests leading to their choice of 6 representatives, the IMO stands as the culminating symbol for the surprisingly expansive and wonderful world which is contest math.
+
+<Figure
+    image="usa_imo.png"
+/>
+
+The contest is essentially a test, split over two days, with three questions given over four and a half hours each day. The questions are all proofs, meaning you don’t simply find some numerical answer, you have to discover and articulate a rigorous line of reasoning to answer each difficult question, and they are scored on a scale from 0 to 7.
+
+<Figure
+    image="imo_q.png"
+/>
+
+Of interest to us today is the one from 2011, with 563 total participants representing 101 countries. I know what you’re thinking, and the answer is yes. Those do all happen to be prime numbers. But that’s not why this test was interesting, out of these prime problem solvers, only one, Lisa Saurmann from Germany, got a perfect score. The only thing standing between the next two runners up and a perfect score that year was problem #2. This problem is beautiful, and despite evading many of the world’s best mathematicians of their age, the solution is something which anyone watching this video can understand.
+
+## The problem
+
```

Comment:
I would start by showing the full question, then breaking it down piece by piece.